style.css

```
@@ -547,3 +547,494 @@ hr {
   border-radius: 10px;
   background-color: black;
 }
+
+.title hr {
+  width: 100%;
+  height: 1px;
+}
+
+/* desktop-version */
+
+@media only screen and (min-width: 760px) {
```

- [ ] Hi so far so good. I would like you to kindly increase the `min-width` from `760px` to something in the range of `992px` - `1100px`. because that would be the correct width for desktop size.

something like this:
```suggestion
@media only screen and (min-width: 992px) {
```

---

Correction has been made